(dp1
S'output'
p2
S"<type 'exceptions.AttributeError'> type object 'SQLFORM' has no attribute 'generator'"
p3
sS'layer'
p4
S'/home/praveeny/web2py/applications/courier_portal/controllers/default.py'
p5
sS'code'
p6
S'\ndef index():\n    form = SQLFORM.generator(db.courier).process()\n    rows = db(db.courier).select()\n    return locals()\n\ndef show():\n    courier_obj=db.courier(request.args(0,cast=int))\n    return locals()\n\ndef user():\n    """\n    exposes:\n    http://..../[app]/default/user/login\n    http://..../[app]/default/user/logout\n    http://..../[app]/default/user/register\n    http://..../[app]/default/user/profile\n    http://..../[app]/default/user/retrieve_password\n    http://..../[app]/default/user/change_password\n    http://..../[app]/default/user/manage_users (requires membership in\n    http://..../[app]/default/user/bulk_register\n    use @auth.requires_login()\n        @auth.requires_membership(\'group name\')\n        @auth.requires_permission(\'read\',\'table name\',record_id)\n    to decorate functions that need access control\n    """\n    return dict(form=auth())\n\nresponse._vars=response._caller(index)\n'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/praveeny/web2py/gluon/restricted.py", line 227, in restricted\n    exec ccode in environment\n  File "/home/praveeny/web2py/applications/courier_portal/controllers/default.py", line 29, in <module>\n  File "/home/praveeny/web2py/gluon/globals.py", line 412, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/praveeny/web2py/applications/courier_portal/controllers/default.py", line 3, in index\n    form = SQLFORM.generator(db.courier).process()\nAttributeError: type object \'SQLFORM\' has no attribute \'generator\'\n'
p11
s.